.travis.yml

```diff
@@ -46,6 +46,10 @@ after_script:
 
 jobs:
   include:
+      - name: "Check Code Style"
+        script:
+         - rustup component add rustfmt
+         - cargo fmt -- --check
```

Comment:
Before running this command, it would be also nice to output the version of rustfmt, just to better understand what Travis CI is currently using. This may help debugging potential issues.